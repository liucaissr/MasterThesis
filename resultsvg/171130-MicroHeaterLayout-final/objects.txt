Chip 25672992.098531
line1: -214.606+27348.597i,4952.221+27348.597i
line2: 4952.221+27348.597i,4952.221+32317.410i
line3: 4952.221+32317.410i,-214.606+32317.410i
line4: -214.606+32317.410i,-214.606+27348.597i
o1 16725.250000
line1: 1056.558+29024.845i,1281.058+29024.845i
line2: 1281.058+29024.845i,1281.058+29099.345i
line3: 1281.058+29099.345i,1056.558+29099.345i
line4: 1056.558+29099.345i,1056.558+29024.845i
o2 16725.250000
line1: 1356.558+29024.845i,1581.058+29024.845i
line2: 1581.058+29024.845i,1581.058+29099.345i
line3: 1581.058+29099.345i,1356.558+29099.345i
line4: 1356.558+29099.345i,1356.558+29024.845i
o3 16725.324500
line1: 1656.557+29024.845i,1881.058+29024.845i
line2: 1881.058+29024.845i,1881.058+29099.345i
line3: 1881.058+29099.345i,1656.557+29099.345i
line4: 1656.557+29099.345i,1656.557+29024.845i
o4 227802.378209
line1: 2106.557+29024.845i,2181.058+29024.845i
line2: 2181.058+29024.845i,2181.058+32082.554i
line3: 2181.058+32082.554i,2106.557+32082.554i
line4: 2106.557+32082.554i,2106.557+29024.845i
o5 16725.250000
line1: 2256.558+29024.845i,2481.058+29024.845i
line2: 2481.058+29024.845i,2481.058+29099.345i
line3: 2481.058+29099.345i,2256.558+29099.345i
line4: 2256.558+29099.345i,2256.558+29024.845i
o6 227802.378209
line1: 2706.557+29024.845i,2781.058+29024.845i
line2: 2781.058+29024.845i,2781.058+32082.554i
line3: 2781.058+32082.554i,2706.557+32082.554i
line4: 2706.557+32082.554i,2706.557+29024.845i
o7 16725.250000
line1: 2856.558+29024.845i,3081.058+29024.845i
line2: 3081.058+29024.845i,3081.058+29099.345i
line3: 3081.058+29099.345i,2856.558+29099.345i
line4: 2856.558+29099.345i,2856.558+29024.845i
o8 16725.250000
line1: 3156.557+29024.845i,3381.057+29024.845i
line2: 3381.057+29024.845i,3381.057+29099.345i
line3: 3381.057+29099.345i,3156.557+29099.345i
line4: 3156.557+29099.345i,3156.557+29024.845i
o9 16725.250000
line1: 3456.558+29024.845i,3681.058+29024.845i
line2: 3681.058+29024.845i,3681.058+29099.345i
line3: 3681.058+29099.345i,3456.558+29099.345i
line4: 3456.558+29099.345i,3456.558+29024.845i
o10 16725.250000
line1: 1206.558+32008.053i,1431.058+32008.053i
line2: 1431.058+32008.053i,1431.058+32082.553i
line3: 1431.058+32082.553i,1206.558+32082.553i
line4: 1206.558+32082.553i,1206.558+32008.053i
o11 16725.250000
line1: 1506.558+32008.054i,1731.058+32008.054i
line2: 1731.058+32008.054i,1731.058+32082.554i
line3: 1731.058+32082.554i,1506.558+32082.554i
line4: 1506.558+32082.554i,1506.558+32008.054i
o12 16725.250000
line1: 1806.558+32008.054i,2031.058+32008.054i
line2: 2031.058+32008.054i,2031.058+32082.554i
line3: 2031.058+32082.554i,1806.558+32082.554i
line4: 1806.558+32082.554i,1806.558+32008.054i
o13 16725.250000
line1: 2406.558+32008.054i,2631.058+32008.054i
line2: 2631.058+32008.054i,2631.058+32082.554i
line3: 2631.058+32082.554i,2406.558+32082.554i
line4: 2406.558+32082.554i,2406.558+32008.054i
o14 16725.250000
line1: 3006.558+32008.054i,3231.058+32008.054i
line2: 3231.058+32008.054i,3231.058+32082.554i
line3: 3231.058+32082.554i,3006.558+32082.554i
line4: 3006.558+32082.554i,3006.558+32008.054i
o15 16725.250000
line1: 3306.558+32008.054i,3531.058+32008.054i
line2: 3531.058+32008.054i,3531.058+32082.554i
line3: 3531.058+32082.554i,3306.558+32082.554i
line4: 3306.558+32082.554i,3306.558+32008.054i
o16 2927573.796000
line1: 270.250+32082.553i,1131.058+32082.553i
line2: 1131.058+32082.553i,1131.058+31583.053i
line3: 1131.058+31583.053i,769.750+31583.053i
line4: 769.750+31583.053i,769.750+28583.053i
line5: 769.750+28583.053i,1019.750+28583.053i
line6: 1019.750+28583.053i,1019.750+27583.453i
line7: 1019.750+27583.453i,20.250+27583.453i
line8: 20.250+27583.453i,20.250+28583.053i
line9: 20.250+28583.053i,270.250+28583.053i
line10: 270.250+28583.053i,270.250+32082.553i
o17 2927573.434692
line1: 3606.557+32082.553i,4467.365+32082.553i
line2: 4467.365+32082.553i,4467.365+28583.053i
line3: 4467.365+28583.053i,4717.365+28583.053i
line4: 4717.365+28583.053i,4717.365+27583.453i
line5: 4717.365+27583.453i,3717.865+27583.453i
line6: 3717.865+27583.453i,3717.865+28583.053i
line7: 3717.865+28583.053i,3967.865+28583.053i
line8: 3967.865+28583.053i,3967.865+31583.054i
line9: 3967.865+31583.054i,3606.557+31583.054i
line10: 3606.557+31583.054i,3606.557+32082.553i
o18 15060.152650
line1: 1956.558+29024.845i,2158.708+29024.845i
line2: 2158.708+29024.845i,2158.708+29099.345i
line3: 2158.708+29099.345i,1956.558+29099.345i
line4: 1956.558+29099.345i,1956.558+29024.845i
o19 15060.152650
line1: 2128.907+32008.054i,2331.057+32008.054i
line2: 2331.057+32008.054i,2331.057+32082.554i
line3: 2331.057+32082.554i,2128.907+32082.554i
line4: 2128.907+32082.554i,2128.907+32008.054i
o20 15060.227150
line1: 2556.557+29024.845i,2758.708+29024.845i
line2: 2758.708+29024.845i,2758.708+29099.345i
line3: 2758.708+29099.345i,2556.557+29099.345i
line4: 2556.557+29099.345i,2556.557+29024.845i
o21 15060.152650
line1: 2728.907+32008.054i,2931.057+32008.054i
line2: 2931.057+32008.054i,2931.057+32082.554i
line3: 2931.057+32082.554i,2728.907+32082.554i
line4: 2728.907+32082.554i,2728.907+32008.054i
o22 224469.096000
line1: 1206.558+29047.195i,1281.058+29047.195i
line2: 1281.058+29047.195i,1281.058+32060.203i
line3: 1281.058+32060.203i,1206.558+32060.203i
line4: 1206.558+32060.203i,1206.558+29047.195i
o23 224469.096000
line1: 1356.558+29047.195i,1431.058+29047.195i
line2: 1431.058+29047.195i,1431.058+32060.203i
line3: 1431.058+32060.203i,1356.558+32060.203i
line4: 1356.558+32060.203i,1356.558+29047.195i
o24 224469.170500
line1: 1506.558+29047.195i,1581.058+29047.195i
line2: 1581.058+29047.195i,1581.058+32060.204i
line3: 1581.058+32060.204i,1506.558+32060.204i
line4: 1506.558+32060.204i,1506.558+29047.195i
o25 224469.170500
line1: 1656.558+29047.195i,1731.058+29047.195i
line2: 1731.058+29047.195i,1731.058+32060.204i
line3: 1731.058+32060.204i,1656.558+32060.204i
line4: 1656.558+32060.204i,1656.558+29047.195i
o26 224469.170500
line1: 1806.558+29047.195i,1881.058+29047.195i
line2: 1881.058+29047.195i,1881.058+32060.204i
line3: 1881.058+32060.204i,1806.558+32060.204i
line4: 1806.558+32060.204i,1806.558+29047.195i
o27 224469.170500
line1: 2406.558+29047.195i,2481.058+29047.195i
line2: 2481.058+29047.195i,2481.058+32060.204i
line3: 2481.058+32060.204i,2406.558+32060.204i
line4: 2406.558+32060.204i,2406.558+29047.195i
o28 224469.170500
line1: 3006.558+29047.195i,3081.058+29047.195i
line2: 3081.058+29047.195i,3081.058+32060.204i
line3: 3081.058+32060.204i,3006.558+32060.204i
line4: 3006.558+32060.204i,3006.558+29047.195i
o29 224469.170500
line1: 3156.558+29047.195i,3231.058+29047.195i
line2: 3231.058+29047.195i,3231.058+32060.204i
line3: 3231.058+32060.204i,3156.558+32060.204i
line4: 3156.558+32060.204i,3156.558+29047.195i
o30 224466.157491
line1: 3306.558+29047.195i,3381.057+29047.195i
line2: 3381.057+29047.195i,3381.057+32060.204i
line3: 3381.057+32060.204i,3306.558+32060.204i
line4: 3306.558+32060.204i,3306.558+29047.195i
o31 224469.170500
line1: 3456.558+29047.195i,3531.058+29047.195i
line2: 3531.058+29047.195i,3531.058+32060.204i
line3: 3531.058+32060.204i,3456.558+32060.204i
line4: 3456.558+32060.204i,3456.558+29047.195i
o32 214970.346000
line1: 1056.558+29047.195i,1131.058+29047.195i
line2: 1131.058+29047.195i,1131.058+31932.703i
line3: 1131.058+31932.703i,1056.558+31932.703i
line4: 1056.558+31932.703i,1056.558+29047.195i
o33 214970.368350
line1: 3606.558+29047.195i,3681.058+29047.195i
line2: 3681.058+29047.195i,3681.058+31932.703i
line3: 3681.058+31932.703i,3606.558+31932.703i
line4: 3606.558+31932.703i,3606.558+29047.195i
o34 224469.170500
line1: 1956.558+29047.195i,2031.058+29047.195i
line2: 2031.058+29047.195i,2031.058+32060.204i
line3: 2031.058+32060.204i,1956.558+32060.204i
line4: 1956.558+32060.204i,1956.558+29047.195i
o35 224466.157491
line1: 2256.558+29047.195i,2331.057+29047.195i
line2: 2331.057+29047.195i,2331.057+32060.204i
line3: 2331.057+32060.204i,2256.558+32060.204i
line4: 2256.558+32060.204i,2256.558+29047.195i
o36 224469.170500
line1: 2556.558+29047.195i,2631.058+29047.195i
line2: 2631.058+29047.195i,2631.058+32060.204i
line3: 2631.058+32060.204i,2556.558+32060.204i
line4: 2556.558+32060.204i,2556.558+29047.195i
o37 224466.157491
line1: 2856.558+29047.195i,2931.057+29047.195i
line2: 2931.057+29047.195i,2931.057+32060.204i
line3: 2931.057+32060.204i,2856.558+32060.204i
line4: 2856.558+32060.204i,2856.558+29047.195i
