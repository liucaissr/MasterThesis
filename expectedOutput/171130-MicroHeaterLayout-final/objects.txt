Chip 25672992.098531
line1: -224.606+11455.695i,4942.221+11455.695i
line2: 4942.221+11455.695i,4942.221+16424.508i
line3: 4942.221+16424.508i,-224.606+16424.508i
line4: -224.606+16424.508i,-224.606+11455.695i
o1 16725.250000
line1: 1046.558+13131.943i,1271.058+13131.943i
line2: 1271.058+13131.943i,1271.058+13206.443i
line3: 1271.058+13206.443i,1046.558+13206.443i
line4: 1046.558+13206.443i,1046.558+13131.943i
o2 16725.250000
line1: 1346.558+13131.943i,1571.058+13131.943i
line2: 1571.058+13131.943i,1571.058+13206.443i
line3: 1571.058+13206.443i,1346.558+13206.443i
line4: 1346.558+13206.443i,1346.558+13131.943i
o3 16725.324500
line1: 1646.557+13131.943i,1871.058+13131.943i
line2: 1871.058+13131.943i,1871.058+13206.443i
line3: 1871.058+13206.443i,1646.557+13206.443i
line4: 1646.557+13206.443i,1646.557+13131.943i
o4 227802.378209
line1: 2096.557+13131.943i,2171.058+13131.943i
line2: 2171.058+13131.943i,2171.058+16189.652i
line3: 2171.058+16189.652i,2096.557+16189.652i
line4: 2096.557+16189.652i,2096.557+13131.943i
o5 16725.250000
line1: 2246.558+13131.943i,2471.058+13131.943i
line2: 2471.058+13131.943i,2471.058+13206.443i
line3: 2471.058+13206.443i,2246.558+13206.443i
line4: 2246.558+13206.443i,2246.558+13131.943i
o6 227802.378209
line1: 2696.557+13131.943i,2771.058+13131.943i
line2: 2771.058+13131.943i,2771.058+16189.652i
line3: 2771.058+16189.652i,2696.557+16189.652i
line4: 2696.557+16189.652i,2696.557+13131.943i
o7 16725.250000
line1: 2846.558+13131.943i,3071.058+13131.943i
line2: 3071.058+13131.943i,3071.058+13206.443i
line3: 3071.058+13206.443i,2846.558+13206.443i
line4: 2846.558+13206.443i,2846.558+13131.943i
o8 16725.250000
line1: 3146.557+13131.943i,3371.057+13131.943i
line2: 3371.057+13131.943i,3371.057+13206.443i
line3: 3371.057+13206.443i,3146.557+13206.443i
line4: 3146.557+13206.443i,3146.557+13131.943i
o9 16725.250000
line1: 3446.558+13131.943i,3671.058+13131.943i
line2: 3671.058+13131.943i,3671.058+13206.443i
line3: 3671.058+13206.443i,3446.558+13206.443i
line4: 3446.558+13206.443i,3446.558+13131.943i
o10 16725.250000
line1: 1196.558+16115.151i,1421.058+16115.151i
line2: 1421.058+16115.151i,1421.058+16189.651i
line3: 1421.058+16189.651i,1196.558+16189.651i
line4: 1196.558+16189.651i,1196.558+16115.151i
o11 16725.250000
line1: 1496.558+16115.152i,1721.058+16115.152i
line2: 1721.058+16115.152i,1721.058+16189.652i
line3: 1721.058+16189.652i,1496.558+16189.652i
line4: 1496.558+16189.652i,1496.558+16115.152i
o12 16725.250000
line1: 1796.558+16115.152i,2021.058+16115.152i
line2: 2021.058+16115.152i,2021.058+16189.652i
line3: 2021.058+16189.652i,1796.558+16189.652i
line4: 1796.558+16189.652i,1796.558+16115.152i
o13 16725.250000
line1: 2396.558+16115.152i,2621.058+16115.152i
line2: 2621.058+16115.152i,2621.058+16189.652i
line3: 2621.058+16189.652i,2396.558+16189.652i
line4: 2396.558+16189.652i,2396.558+16115.152i
o14 16725.250000
line1: 2996.558+16115.152i,3221.058+16115.152i
line2: 3221.058+16115.152i,3221.058+16189.652i
line3: 3221.058+16189.652i,2996.558+16189.652i
line4: 2996.558+16189.652i,2996.558+16115.152i
o15 16725.250000
line1: 3296.558+16115.152i,3521.058+16115.152i
line2: 3521.058+16115.152i,3521.058+16189.652i
line3: 3521.058+16189.652i,3296.558+16189.652i
line4: 3296.558+16189.652i,3296.558+16115.152i
o16 2927573.796000
line1: 260.250+16189.651i,1121.058+16189.651i
line2: 1121.058+16189.651i,1121.058+15690.151i
line3: 1121.058+15690.151i,759.750+15690.151i
line4: 759.750+15690.151i,759.750+12690.151i
line5: 759.750+12690.151i,1009.750+12690.151i
line6: 1009.750+12690.151i,1009.750+11690.551i
line7: 1009.750+11690.551i,10.250+11690.551i
line8: 10.250+11690.551i,10.250+12690.151i
line9: 10.250+12690.151i,260.250+12690.151i
line10: 260.250+12690.151i,260.250+16189.651i
o17 2927573.434692
line1: 3596.557+16189.651i,4457.365+16189.651i
line2: 4457.365+16189.651i,4457.365+12690.151i
line3: 4457.365+12690.151i,4707.365+12690.151i
line4: 4707.365+12690.151i,4707.365+11690.551i
line5: 4707.365+11690.551i,3707.865+11690.551i
line6: 3707.865+11690.551i,3707.865+12690.151i
line7: 3707.865+12690.151i,3957.865+12690.151i
line8: 3957.865+12690.151i,3957.865+15690.152i
line9: 3957.865+15690.152i,3596.557+15690.152i
line10: 3596.557+15690.152i,3596.557+16189.651i
o18 15060.152650
line1: 1946.558+13131.943i,2148.708+13131.943i
line2: 2148.708+13131.943i,2148.708+13206.443i
line3: 2148.708+13206.443i,1946.558+13206.443i
line4: 1946.558+13206.443i,1946.558+13131.943i
o19 15060.152650
line1: 2118.907+16115.152i,2321.057+16115.152i
line2: 2321.057+16115.152i,2321.057+16189.652i
line3: 2321.057+16189.652i,2118.907+16189.652i
line4: 2118.907+16189.652i,2118.907+16115.152i
o20 15060.227150
line1: 2546.557+13131.943i,2748.708+13131.943i
line2: 2748.708+13131.943i,2748.708+13206.443i
line3: 2748.708+13206.443i,2546.557+13206.443i
line4: 2546.557+13206.443i,2546.557+13131.943i
o21 15060.152650
line1: 2718.907+16115.152i,2921.057+16115.152i
line2: 2921.057+16115.152i,2921.057+16189.652i
line3: 2921.057+16189.652i,2718.907+16189.652i
line4: 2718.907+16189.652i,2718.907+16115.152i
o22 224469.096000
line1: 1196.558+13154.293i,1271.058+13154.293i
line2: 1271.058+13154.293i,1271.058+16167.301i
line3: 1271.058+16167.301i,1196.558+16167.301i
line4: 1196.558+16167.301i,1196.558+13154.293i
o23 224469.096000
line1: 1346.558+13154.293i,1421.058+13154.293i
line2: 1421.058+13154.293i,1421.058+16167.301i
line3: 1421.058+16167.301i,1346.558+16167.301i
line4: 1346.558+16167.301i,1346.558+13154.293i
o24 224469.170500
line1: 1496.558+13154.293i,1571.058+13154.293i
line2: 1571.058+13154.293i,1571.058+16167.302i
line3: 1571.058+16167.302i,1496.558+16167.302i
line4: 1496.558+16167.302i,1496.558+13154.293i
o25 224469.170500
line1: 1646.558+13154.293i,1721.058+13154.293i
line2: 1721.058+13154.293i,1721.058+16167.302i
line3: 1721.058+16167.302i,1646.558+16167.302i
line4: 1646.558+16167.302i,1646.558+13154.293i
o26 224469.170500
line1: 1796.558+13154.293i,1871.058+13154.293i
line2: 1871.058+13154.293i,1871.058+16167.302i
line3: 1871.058+16167.302i,1796.558+16167.302i
line4: 1796.558+16167.302i,1796.558+13154.293i
o27 224469.170500
line1: 2396.558+13154.293i,2471.058+13154.293i
line2: 2471.058+13154.293i,2471.058+16167.302i
line3: 2471.058+16167.302i,2396.558+16167.302i
line4: 2396.558+16167.302i,2396.558+13154.293i
o28 224469.170500
line1: 2996.558+13154.293i,3071.058+13154.293i
line2: 3071.058+13154.293i,3071.058+16167.302i
line3: 3071.058+16167.302i,2996.558+16167.302i
line4: 2996.558+16167.302i,2996.558+13154.293i
o29 224469.170500
line1: 3146.558+13154.293i,3221.058+13154.293i
line2: 3221.058+13154.293i,3221.058+16167.302i
line3: 3221.058+16167.302i,3146.558+16167.302i
line4: 3146.558+16167.302i,3146.558+13154.293i
o30 224466.157491
line1: 3296.558+13154.293i,3371.057+13154.293i
line2: 3371.057+13154.293i,3371.057+16167.302i
line3: 3371.057+16167.302i,3296.558+16167.302i
line4: 3296.558+16167.302i,3296.558+13154.293i
o31 224469.170500
line1: 3446.558+13154.293i,3521.058+13154.293i
line2: 3521.058+13154.293i,3521.058+16167.302i
line3: 3521.058+16167.302i,3446.558+16167.302i
line4: 3446.558+16167.302i,3446.558+13154.293i
o32 214970.346000
line1: 1046.558+13154.293i,1121.058+13154.293i
line2: 1121.058+13154.293i,1121.058+16039.801i
line3: 1121.058+16039.801i,1046.558+16039.801i
line4: 1046.558+16039.801i,1046.558+13154.293i
o33 214970.368350
line1: 3596.558+13154.293i,3671.058+13154.293i
line2: 3671.058+13154.293i,3671.058+16039.801i
line3: 3671.058+16039.801i,3596.558+16039.801i
line4: 3596.558+16039.801i,3596.558+13154.293i
o34 224469.170500
line1: 1946.558+13154.293i,2021.058+13154.293i
line2: 2021.058+13154.293i,2021.058+16167.302i
line3: 2021.058+16167.302i,1946.558+16167.302i
line4: 1946.558+16167.302i,1946.558+13154.293i
o35 224466.157491
line1: 2246.558+13154.293i,2321.057+13154.293i
line2: 2321.057+13154.293i,2321.057+16167.302i
line3: 2321.057+16167.302i,2246.558+16167.302i
line4: 2246.558+16167.302i,2246.558+13154.293i
o36 224469.170500
line1: 2546.558+13154.293i,2621.058+13154.293i
line2: 2621.058+13154.293i,2621.058+16167.302i
line3: 2621.058+16167.302i,2546.558+16167.302i
line4: 2546.558+16167.302i,2546.558+13154.293i
o37 224466.157491
line1: 2846.558+13154.293i,2921.057+13154.293i
line2: 2921.057+13154.293i,2921.057+16167.302i
line3: 2921.057+16167.302i,2846.558+16167.302i
line4: 2846.558+16167.302i,2846.558+13154.293i
